### PR TITLE
Add test for defining both a start and end point for path

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- start and endpoint already work together 
- add test to ensure that paths will work when start and end point are both provided
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by EDsCODE** [comment]
changing to WIP. something doesn't seem right

File: ee/clickhouse/queries/paths/paths.py [hunk 1]
```diff
@@ -94,18 +88,49 @@ def get_path_query_by_funnel(self, funnel_filter: Filter):
...
+
+        if not conditions:
+            return "", params
+
+        return f"WHERE {' AND '.join(conditions)}", params
+
+    def get_target_clause(self) -> str:
+        if self._filter.end_point and self._filter.start_point:
+            return """
+            , indexOf(compact_path, %(secondary_target_point)s) as start_target_index
```
Comment (neilkakkar) [inline ee/clickhouse/queries/paths/paths.py:116]: This makes sense to me.

An alternative here might be to always keep these in SQL, and just update params for `target_point` and `secondary_target_point` ? Seems easier to read & use, at a very minimal cost of time-complexity. Thoughts?

Comment (EDsCODE) [reply ee/clickhouse/queries/paths/paths.py:116]: wouldn't need the first half of the clause if we only have one target though

**Comment by EDsCODE** [comment]
The one problem is defining a # of events limit when defining both start and end point

**Comment by neilkakkar** [comment]
> The one problem is defining a # of events limit when defining both start and end point 

Right, that makes sense. This is actually what I aimed to solve in the metabase query, with: 

```
if(length(paths) > 6, arrayConcat(arraySlice(paths, 1, 3), ['...'], arraySlice(paths, -3, 3)), paths) AS paths
```

The idea being, choose `limit/2` from start, `limit/2` from end, and aggregate the middle into "others"

File: ee/clickhouse/queries/paths/paths.py (hunk 1 above)
Comment (neilkakkar) [reply ee/clickhouse/queries/paths/paths.py:106]: Yeah, the idea was that even if we don't need it, the "start index" value would be -1, which means the if conditional would disregard it.

So we wouldn't need to do any modifications during query build time, and all things would be taken care of during execution time.

It's a minor point though, I'm okay with either right now, we'll find out more once we add more features here and see how this fares.

**Review by neilkakkar — APPROVED** [review]

**Pull request merged**